### PR TITLE
feat(app-platform): adds instructions for Amixir, ClickUp, and Split

### PR DESCRIPTION
**Pull request by scefali**

### CODE REVIEW

File: src/collections/_documentation/workflow/integrations/global-integrations.md [hunk 1]
```diff
@@ -889,3 +870,66 @@ Now your Slack integration is available to all projects in your Sentry organizat
 #### Deleting the legacy Slack integration
 
 Once you configure the global Slack integration and Alert Rules, you can disable the old Slack integration. You’ll need to go to each project that has it enabled and disable it. We recommend disabling the legacy integration after setting up the global integration.
+
+
+
+## Other
```
Comment (MimiDumpling) [inline src/collections/_documentation/workflow/integrations/global-integrations.md]: What does "Other" mean? Can we give this a more descriptive heading?

File: src/collections/_documentation/workflow/integrations/global-integrations.md [hunk 2]
```diff
@@ -889,3 +870,66 @@ Now your Slack integration is available to all projects in your Sentry organizat
...
+
+{% include components/alert.html
+  title="Note"
+  content="The following integrations **won't** work with self-hosted Sentry."
+  level="warning"
+%}
+
+### Amixr
+
+In Amixr, issues from Sentry get stored as well as alerts from other sources like Grafana or Alertmanager. The Amixr integration synchronizes issue statuses between Amixr and Sentry. Issues get posted to your Slack and users can change the statuses of those issues by clicking on buttons within the message.
```
Comment (MimiDumpling) [inline src/collections/_documentation/workflow/integrations/global-integrations.md]:
```suggestion
In Amixr, issues from Sentry get stored as well as alerts from other sources like Grafana or Alertmanager. The Amixr integration synchronizes issue statuses between Amixr and Sentry. Issues get posted to your Slack, and users can change the statuses of those issues by clicking on buttons within the message.
```

File: src/collections/_documentation/workflow/integrations/global-integrations.md [hunk 3]
```diff
@@ -889,3 +870,66 @@ Now your Slack integration is available to all projects in your Sentry organizat
...
+3. Clicking on the Rookout icon takes you to Rookout's web UI where you can continue the debugging process. Rookout makes the best guess for the corresponding project and file in its web UI and will take you to the correct line of code in the file.
+
+    [{% asset rookout/rookout_screen_1.png alt="Sentry modal indicating that permissions need installation." %}]({% asset rookout/rookout_screen_1.png @path %})
+
+
+### Split
+
+The Split integration quickly processes and displays Sentry exception data in the Split platform as track events for analysis. You can control what environments and traffic types you're capturing exceptions for in the Split dashboard without having to touch any code.
+
+The Split integration is only available for organizations on the Busines and Enterprise plans.
```
Comment (MimiDumpling) [inline src/collections/_documentation/workflow/integrations/global-integrations.md]:
```suggestion
The Split integration is only available for organizations on the Business and Enterprise plans.
```

File: src/collections/_documentation/workflow/integrations/global-integrations.md (hunk 1 above)
Comment (scefali) [reply src/collections/_documentation/workflow/integrations/global-integrations.md]: @MimiDumpling Other means it doesn't fit into the other categories. Should I say that?

**Comment by scefali** [comment]
@MimiDumpling As requested, I made new sections.

**Pull request merged**